spec/features/user_can_do_things_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'rails_helper'
+
+describe 'as a user' do
+  describe 'when logged in' do
+    it 'shows things on dashboard' do
+      user_1 = create(:user, github_token: ENV["GITHUB_TOKEN"])
+      token_1 = user_1.github_token
+
+      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user_1)
+
+      visit '/dashboard'
+      expect(page).to have_content('Github')
+
+      within('.github-list') do
+        expect(page).to have_css('.repository', count: 5)
+      end
+
+      within('.following-list') do
+        expect(page).to have_css('.following', count: 5)
+      end
+
+      within('.followers-list') do
+        expect(page).to have_css('.follower', count: 5)
+      end
+    end
+
+    it 'can log in with github' do
+      stub_omniauth
+      user_1 = create(:user)
```

Comment:
It may not matter much, but it makes more sense to me to move stub_omniauth below the user creation, but again, if it's not affecting tests, I'm fine with this.